backend/clubs/views.py

```diff
@@ -5023,24 +5028,44 @@ def initiate_checkout(self, request, *args, **kwargs):
         )
 
         # Assert that the filter succeeded in freezing all the tickets for checkout
-        if tickets.count() != cart.tickets.count():
+        if tickets.count() != cart.tickets.all().count() or tickets.count() == 0:
```

Comment:
We're checking `if not cart.tickets.exists()` in L5014 -- second condition is unnecessary.